src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3883,6 +3886,15 @@ private bool SelfDescribingEvents
         internal const string s_ActivityStartSuffix = "Start";
         internal const string s_ActivityStopSuffix = "Stop";
 
+        internal const string MultipleEventSourcesSwitch = "System.Diagnostics.Tracing.EventSource.AllowMultipleEventSources";
+        internal const string MultipleEventSourcesEnvironmentVariable = "DOTNET_" + MultipleEventSourcesSwitch;
```

Comment:
This will be very unusual env variable name. Do we really need env variable to enable this? I would thin that the appcontext switch should be enough in this case.

Reply:
+1. From the email discussion I also had the impression no env var was needed and they are happy to set a switch. You could certainly with them though.

Reply:
Sure, I can reduce it to just the config switch. I think there was a helper class that would even cache the value in that case.